packages/dht/protos/DhtRpc.proto

```diff
@@ -102,11 +103,13 @@ message ClosestPeersResponse {
   string requestId = 2;
 }
 
+// TODO rename to ClosestRingContactsRequest
 message ClosestRingPeersRequest {
```

Comment:
These message names and the names of their fields could be renamed without it being a breaking breaking change. But could keep them as is for consistency